portfolio.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="portfolio.css">
+</head>
+
+<body>
+    <section class="background">
+        <div class="toolbar">
+            <div class="welcome">Welcome</div>
+            <img class="menu" src="./assets/menu.svg" alt="menu">
+        </div>
+        <!-- <div style="display: flex; justify-content: center;">
+            <p class="margin-10px" style="cursor: pointer;">Portfolio</p>
+            <p class="margin-10px" style="cursor: pointer;">About</p>
+            <p class="margin-10px" style="cursor: pointer;">Contact</p>
+        </div> -->
```

Comment:
- [ ] Remove the commented code from your project as it makes code hard to read.